recipe/meta.yaml

```diff
@@ -11,21 +11,17 @@ source:
 
 
 build:
+  noarch: python
   number: 0
   script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
-  build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    # Minor version must match between antlr and antlr-python-runtime
-    # https://github.com/omry/omegaconf/issues/798
-    - antlr 4.9
   host:
-    - python
+    - python >=3.6
     - pip
+    - antlr 4.9
   run:
-    - python
+    - python >=3.6
```

Comment:
```suggestion
    - python >=3.7
```

Reply:
I generally follow whatever pins are declared upstream unless there's a particularly good reason to deviate from them. Others have differing opinions. I don't feel strongly since this is a fairly rare edge case.

Reply:
Actually, there aren't any Python 3.6 builds for `omegaconf=2.2.2` at https://anaconda.org/conda-forge/omegaconf/files, so... do we want to add back support for it? :laughing:

Reply:
You've convinced me. :smile: